tests/core/full_node/test_mempool.py

```diff
@@ -1510,7 +1516,6 @@ async def test_double_spend_same_bundle(self, bt, two_nodes_one_block, wallet_a)
             farmer_reward_puzzle_hash=reward_ph,
             pool_reward_puzzle_hash=reward_ph,
         )
-        peer = await connect_and_get_peer(server_1, server_2, bt.config["self_hostname"])
```

Comment:
Are we certain we're not just doing this for the side-effect? (Although if the test still passes, I guess it doesn't really matter.)

Reply:
I suspect it was left-over from copy-paste